src/routes/docs/widget/tutorials/+page.md

```diff
@@ -7,3 +11,49 @@ Below are tutorials for popular Web frameworks/libraries, like React, Angular an
 ### Angular with Angular CLI
 
 ### Modal with Vue and Vite
+
+### "Magic links" and journey continuation
+
+Verifying a user's email address, building a "forgot password" flow, or using an email as a two-factor authentication method, ForgeRock's Email Suspend Node within journeys is the tool for the job. This results in the journey/tree being suspended until the user clicks a link (aka "magic link") found in their email. This link contains a randomly generated code that is used to continue the suspended journey.
+
+Here's how to take advantage of this feature:
+
+First, add the Email Suspend Node to the portion of the journey you'd like to suspend until the user continues the journey from the link found in their email.
```

Comment:
I think you could save yourself a screenshot and provide more context here by linking to the Backstage docs for Suspended Authentication - there is an example tree there too:

https://backstage.forgerock.com/docs/am/7/authentication-guide/authn-suspended.html#authn-suspended

Reply:
Agreed. I'll be cleaning up the docs for the rest of this sprint, so I'll add this to the small task list I have locally. Since I have two approvals and a green build, I'll save our CI from having to run a full build just for a small change to the doc.